modules/op-release-notes-1-7.adoc

```diff
@@ -411,3 +411,20 @@ time="2022-03-04T09:47:57Z" level=error msg="(unable to determine exit status)"
 +
 With this update, the `pipelines-scc` security context constraint (SCC) is compatible with the `SETFCAP` capability necessary for S2I and Buildah clustertasks. As a result, the S2I build tasks run successfully.
 // https://issues.redhat.com/browse/SRVKP-2091
+
+
+[id="release-notes-1-7-3_{context}"]
+== Release notes for {pipelines-title} General Availability 1.7.3
+
+With this update, {pipelines-title} General Availability (GA) 1.7.3 is available on {product-title} 4.9, 4.10, and 4.11.
+
+[id="fixed-issues-1-7-3_{context}"]
+=== Fixed issues
+
+* Before this update, the Operator failed when creating RBAC resources if any namespace was in a `Terminating` state. With this update, the Operator ignores namespaces in a `Terminating` state and creates the RBAC resources.
+// link:https://issues.redhat.com/browse/SRVKP-2248
+// Shivam Mukhade
+
+* Previously, upgrading the {pipelines-title} Operator caused the `pipeline` service account to be recreated, which meant that the secrets linked to the service account were lost. This update fixes the issue. During upgrades, the Operator no longer recreates the `pipeline` service account. As a result, secrets attached to the `pipeline` service account persist after upgrades, and the resources (tasks and pipelines) continue to work correctly.
```

Comment:
Vale suggests changing "Previously" to "Before this update", but maybe it is just as well to leave it as it is in 1.8, rather than changing it here